pyproject.toml

```diff
@@ -44,6 +44,7 @@ classifiers = [
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
```

Comment:
Currently, asyncpg does not provide binary wheels for CPython 3.13, and building it [fails on GHA](https://github.com/crate/pytest-cratedb/actions/runs/11408347462/job/31746314991?pr=14#step:4:480).

Reply:
Here we go.

- https://github.com/MagicStack/asyncpg/pull/1192

Reply:
[asyncpg 0.30.0](https://github.com/MagicStack/asyncpg/releases/tag/v0.30.0) has been released. Thanks a stack, @elprans and all contributors.

-- https://pypi.org/project/asyncpg/